classes/output/core_renderer.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace theme_citricityxund\output;
+
+use context_course;
+use core_auth\output\login;
+use core_course_list_element;
+use stdClass;
+use html_writer;
+
+class core_renderer extends \theme_boost\output\core_renderer {
+    /**
+     * Renders the login form.
+     *
+     * @param \core_auth\output\login $form The renderable.
+     * @return string
+     */
+    public function render_login(login $form): string {
+        global $SITE;
+        $context = $form->export_for_template($this);
+        $context->sitename = format_string($SITE->fullname, true, ['context' => context_course::instance(SITEID), "escape" => false]);
```

Comment:
don't mix quotation types - escape should really be 'escape'

Reply:
NOTE: It is acceptable to mix quotation types providing the double quotation has variable interpolation - e.g. "escape $somevar"